client/app/dedicated/server/reverse-dns/update/reverse-dns-update.html

```diff
@@ -0,0 +1,29 @@
+<div class="oui-modal oui-modal_shadow">
```

Comment:
I feel like I've seen this 3 times already in the PR, can't this be a Component then ?